evgMrmApp/src/evgMrm.cpp

```diff
@@ -70,6 +85,36 @@ EVRMRM::Config evm_evrd_conf = {
     8,  // FP inputs
 };
 
+static
+EVRMRM::Config vme_evm_evru_conf = {
+    "VME-EVM-300 (EVRU)",
```

Comment:
The duplication here and below, just to get a different string, seems excessive.  Maybe just change the strings to `"EVRD"` and `"EVRU"`?

Reply:
I assume you mean change `vme_evm_evru_conf{}` and `vme_evm_evrd_conf{}`. Then `mtca_evm_evru_conf{} `and `mcta_evm_evrd_conf{}` should also change. This is where I got the idea from.

Reply:
Meaning to keep only `evm_evrd_conf` and `evm_evru_conf` and use them for both EVM variants.  Simply change the label strings to read only `"EVRD"` and `"EVRU"`.

Reply:
OK, got it.